test/core/Application.js

```diff
@@ -66,4 +66,68 @@ describe('PIXI.Application', function ()
 
         app.destroy(true);
     });
+
+    describe('set ticker', function ()
+    {
+        const self = this;
```

Comment:
@bigtimebuddy 
in this case I will use `it('<test name>', () =>` instead of `it('<test name>', function ()`, in order to use `this` inside a test. Is it ok?

Reply:
Actually in Mocha it’s all the same context, whether inside it, or containing describe.

Reply:
ok, cool! didn't know, thank you, fixed it